src/SingleBookingPage/bookingItem/AccordionBodyLeg.js

```diff
@@ -0,0 +1,124 @@
+// @flow
```

Comment:
I would suggest one more thing - let's merge these two components together, `AccordionBodyLastLeg` and `AccordionBodyLastLeg` and pass nextLeg={null} for the last one

btw `Kiwi.com Guarantee` needs to be displayed based on data from GraphQL (new field needs to be added) - so let's just render it everytime now

Reply:
> then you don't need to pass down toggle callback

I think the data extracted via their fragments is quite different from one case to the other. If we want to be strict about relay, shouldn't we keep them separate as to not over-request/feed data to the component? (definitely not a critical case, just saying :slightly_smiling_face: )

Reply:
Now I don't know how it's related to relay 🤔